src/xenia/cpu/xex_module.cc

```diff
@@ -281,6 +281,34 @@ bool XexModule::Load(const std::string& name, const std::string& path,
     }
   }
 
+  // Setup memory protection.
+  // TODO: This introduces a load of constants into the JIT, and Xenia isn't
```

Comment:
Over half of x64_sequences.cc isn't setup to handle constants. We probably need a rewrite or something.